src/MessageContainer.js

```diff
@@ -104,10 +104,9 @@ export default class MessageContainer extends React.PureComponent {
           style={styles.listStyle}
           contentContainerStyle={styles.contentContainerStyle}
           renderItem={this.renderRow}
-          renderHeader={this.renderFooter}
-          renderFooter={this.renderLoadEarlier}
           {...this.props.invertibleScrollViewProps}
           ListFooterComponent={this.renderHeaderWrapper}
+          ListHeaderComponent={this.renderFooter}
```

Comment:
Am i mistaken or are the properties swapped? Shouldn't  ListFooterComponent={this.renderFooter}?

Reply:
the list is inverted, so header becomes footer and vice versa. So they are swapped on purpose